packages/react-components/src/styles/index.ts

```diff
@@ -498,6 +498,10 @@ export default createGlobalStyle<Props & ThemeProps>(({ theme, uiHighlight }: Pr
     }
   }
 
+  .opacity-70 {
```

Comment:
Is this actually used anywhere?

Reply:
it's applied to flags when editing 
![image](https://user-images.githubusercontent.com/30404388/131840288-a7b30899-e737-463e-bd0a-5b531547231e.png)

Reply:
Not really, looks like it's styled directly there: https://github.com/polkadot-js/apps/pull/6049/files#diff-288169aeb02a8eaf9159637d338b805e1fa0f48492cc2d5abcac2b72ad0f4747R214.

Reply:
✅